avm/res/compute/ssh-public-key/main.bicep

```diff
@@ -70,7 +70,7 @@ resource avmTelemetry 'Microsoft.Resources/deployments@2024-03-01' = if (enableT
   }
 }
 
-resource sshPublicKey 'Microsoft.Compute/sshPublicKeys@2023-07-01' = {
```

Comment:
No property update required?

Reply:
Nope the public key is still provided and the type is figured during transit.